src/SFA.DAS.ApprenticePortal.SharedUi/Views/Shared/_Footer.cshtml

```diff
@@ -5,8 +5,14 @@
             <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
                 <h2 class="govuk-visually-hidden">Support links</h2>
                 <ul class="govuk-footer__inline-list">
+                    <li class="govuk-footer__inline-list-item">                        
+                        <a class="govuk-footer__link" href="/Privacy">Privacy</a>
```

Comment:
These will work when rendered as part of the Portal website, as the relative URL will be right for the portal.  When rendered from the Shared UI in the Commitments Web the relative URL will end up being "confirm.../Privacy", which won't exist.

There is a helper tag `asp-external-section` that takes care of this:
```suggestion
                        <a class="govuk-footer__link" asp-external-section="@NavigationSection.Home" asp-external-page="Privacy">Privacy</a>
```

Reply:
Updated to use NavigationSection